app/konnect/configure/runtime-manager/runtime-groups/manage.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Manage Runtime Groups
+no_version: true
+---
+
+Create, update, and delete runtime groups through the
+[Runtime Manager](https://konnect.konghq.com/runtime-manager) in Konnect.
+
+You can find the list of all runtime groups in the Runtime Manager. If you don't
+have an Enterprise account, the Runtime Manager will only ever have one
+runtime group entry.
+
+From a runtime group details page, you can
+[create and manage runtime instances](/konnect/configure/runtime-manager/#types-of-runtimes)
+and configure any [global entities](/konnect/configure/runtime-manager/manage-entities)
+in the group.
+
+## Prerequisites
+* TBA: Required permissions
+* TBA: Any prereqs for deleting the group
```

Comment:
* What are the prerequisites for deleting a group? Should you reassign users?
  * What happens to the users that were previously assigned to the group? Are
  they dumped into the default runtime group, or do they have no permissions to
  access anything until manually changed?

Reply:
Per an earlier comment above:
Users exist independently in Konnect. Access to a RG can be assigned to a user by giving the user a specific Role to the RG.

> What are the prerequisites for deleting a group?
None, but the user should beware of the consequences of deleting a RG.
- Existing DPs managed by this RG not be disabled by the deletion of the RG. DPs must be terminated manually by the user.
- All the associated config in the RG will be deleted including associated services, routes, consumers, plugins etc.
- Suggest backing up the RG config with decK before deletion.